tests/nasa/requests/journey_test.py

```diff
@@ -0,0 +1,12 @@
+from pyhafas import HafasClient
+from pyhafas.profile import NASAProfile
+from pyhafas.types.fptf import Journey
+
+
+# Test journey request with NASA profile
+def test_nasa_journey_request():
+    client = HafasClient(NASAProfile())
+    journey = client.journey(
+        journey="¶HKI¶T$A=1@O=Magdeburg, Universitätsbibliothek@L=7482@a=128@$A=1@O=Magdeburg, Neue Str./Zirkusmuseum@L=7411@a=128@$202202271954$202202272011$Str    2$$1$$$$¶KCC¶I1ZFIzEjRVJHIzEjSElOIzAjRUNLIzE4NDc0fDE4NDc0fDE4NDkxfDE4NDkxfDB8MHw1fDE4NDYxfDF8LTIxNDc0ODM2NDZ8MCNHQU0jMjcwMjIyMTk1NCMKWiNWTiMxI1NUIzE2NDU3ODQ5NjEjUEkjMCNaSSMyMzg3I1RBIzQjREEjMjcwMjIyIzFTIzc0NzgjMVQjMTk0OCNMUyM3NTQwI0xUIzIwMjYjUFUjODAjUlQjMSNDQSNTdE4jWkUjMiNaQiNTdHIgICAgMiNQQyM1I0ZSIzc0ODIjRlQjMTk1NCNUTyM3NDExI1RUIzIwMTEj",
```

Comment:
For this to work we need to have a newer journey. I would suggest 
```¶HKI¶T$A=1@O=Magdeburg Hbf@L=8010224@a=128@$A=1@O=Halle (Saale) Hbf@L=8010159@a=128@$202307071735$202307071841$    RE30$$1$$$$$$¶KC¶#VE#2#CF#100#CA#0#CM#0#SICT#0#AM#49#AM2#0#RT#15#¶KCC¶I1ZFIzEjRVJHIzEjSElOIzAjRUNLIzE0MDE1fDE0MDE1fDE0MDgxfDE0MDgxfDB8MHw1fDE0MDEwfDF8MHwyfDB8MHwtMjE0NzQ4MzY0OCNHQU0jNzA3MjMxNzM1IwpaI1ZOIzEjU1QjMTY4ODcxMzcxNSNQSSMwI1pJIzYyNTA3I1RBIzAjREEjNzA3MjMjMVMjODAxMDIyNCMxVCMxNzM1I0xTIzgwMTAxNTkjTFQjMTg0MSNQVSM4MCNSVCMxI0NBI1JFI1pFI1JFMzAjWkIjICAgIFJFMzAjUEMjMyNGUiM4MDEwMjI0I0ZUIzE3MzUjVE8jODAxMDE1OSNUVCMxODQxIw==¶KRCC¶#VE#1#```
(i can also change this, if you want).

Reply:
I updated the journey. Apparently they now use a different format for the station id now. Previously it was just the four digit id like **7482**, now its more complex like: 300**7482**02.